files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -128,38 +128,38 @@ <h3 id="¿Qué_hay_que_hacer_y_en_qué_orden_para_alcanzar_mis_metas">¿Qué hay
     <ol>
      <li>Escribe algunos artículos para iniciar la discusión</li>
      <li>Define cómo deben lucir los artículos</li>
-     <li>Publica los artículos en el sitio web (¿Cómo hacer esto?)</li>
+     <li>Publica esos artículos en el sitio web (¿Cómo hacer esto?)</li>
```

Comment:
* "esos" lleva a la pregunta "cuáles"

```suggestion
     <li>Publica los artículos en el sitio web (¿Cómo hacer esto?)</li>
```